lib/matplotlib/artist.py

```diff
@@ -961,6 +980,21 @@ def format_cursor_data(self, data):
             data = [data]
         return ', '.join('{:0.3g}'.format(item) for item in data)
 
+    @property
+    def mouseover(self):
```

Comment:
Just wondering, have you ever measured the performance of descriptor access vs attribute lookup. You might be quite shocked :scream: